src/IntegrationUtils.ts

```diff
@@ -24,20 +24,26 @@ import * as child_process from 'child_process';
 import { getConfigMaps, getSecrets } from './ConfigMapAndSecrets';
 import * as k8s from 'vscode-kubernetes-tools-api';
 
+const devModeIntegration : string = 'Dev Mode - Apache Camel K Integration in Dev Mode'
 const basicIntegration : string = 'Basic - Apache Camel K Integration (no ConfigMap or Secret)';
 const configMapIntegration : string = 'ConfigMap - Apache Camel K Integration with Kubernetes ConfigMap';
 const secretIntegration : string = 'Secret - Apache Camel K Integration with Kubernetes Secret';
 
 export async function startIntegration(context: vscode.Uri): Promise<any> {
 	return new Promise <any> ( async (resolve, reject) => {
-		await vscode.window.showQuickPick([ basicIntegration, configMapIntegration, secretIntegration ],
+		await vscode.window.showQuickPick([ devModeIntegration, basicIntegration, 
+			configMapIntegration, secretIntegration ],
 		{
 			placeHolder: 'Select the type of Apache Camel K Integration'
 		}).then( async (choice) => {
 			let selectedConfigMap = undefined;
 			let selectedSecret = undefined;
+			let devMode : boolean = false;
```

Comment:
Maybe that's something we could add a setting for? The default should be first in the dropdown list, so perhaps we add a way for the user to customize their environment a bit and use the default that works best for them?

Reply:
ok seems a good way to avoid to really find which option the users are likely to use.

not sure what is easiest to code with VS Code: a user preference or reusing by default the previously selected value for the same command.
Both options seems fine

Reply:
My problem with rolling them together is that they may have some integrations that are in development they want in --dev mode and others that are further along they want to deploy using basic or one of the other approaches NOT in --dev mode. I'd like to not mix them by hiding the option in settings to turn --dev mode on/off for Basic mode, because that seems even more buried.

Reply:
Agree with this single param at a time command palette, it might be too complicated.
I think we will have to work on something to handle a mix of these properties: https://github.com/camel-tooling/vscode-camelk/issues/95